js/apps/admin-ui/src/realm-settings/user-profile/attribute/AttributeGeneralSettings.tsx

```diff
@@ -175,7 +175,7 @@ export const AttributeGeneralSettings = ({
           translationKey={
             editMode
               ? attributeDisplayName
-              : "${profile.attributes." + `${newAttributeName}}`
+              : "profile.attributes." + `${newAttributeName}`
```

Comment:
Let's use string interpolation for the entire string.
```suggestion
              : `profile.attributes.${newAttributeName}`
```